app/models/email.rb

```diff
@@ -0,0 +1,19 @@
+class Email < ApplicationRecord
+  # allow this model to be used as a contact
+  include OwnerContactMap
+  
+  validates :value, presence: true, uniqueness: {case_sensitive: false}, format: { with: /\A[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,}\z/i }
```

Comment:
What happens if two contacts legitimately share the same email address? Like a business partnership where two people share one email address yet have separate phone numbers, or a Braven site where the contact is the same as the person who is the site director, or I'm sure we can imagine other real world scenarios.

If this row can have only one owner and it has a uniqueness constraint, it seems like that would be impossible to model.

Reply:
@adamdruppe @sadleb I've removed the unique value constraint from e-mails, both at the DB and ActiveRecord layers. Thanks for the input, Adam!